### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by Sun-Yiming**
I changed the if condition  at the line 41

### CODE REVIEW

**Comment by pacohung2** [comment]
ok

**Review by pacohung2 — COMMENTED** [review]
very good change :)

**Review by linyuhan98 — COMMENTED** [review]
Good

File: mcr.cpp [hunk 1]
```diff
@@ -20,9 +20,10 @@ int main(){
 	char game[3][3] = {' '}; // Tic-tac-toe
 	char player1 = 'X';
 	char player2 = 'O';
+	int n;
 	bool turn = true; // false for player 1's turn, true for player 2's turn. Player 1 first.
 	cout << "X = Player 1" << endl << "O = Player 2" << endl;
-       for (int n=0; n<9; n++){
```
Comment (kennethzw) [inline mcr.cpp:25]: it is not necessary to remove 'int' out of  'for' ?

**Review by pacohung2 — CHANGES_REQUESTED** [review]
- [x] ???

**Review by kennethzw — COMMENTED** [review]
...

**Pull request merged**